lib/styles.js

```diff
@@ -28,19 +29,28 @@ function StencilStyles() {
             };
 
             this.files = options.files || {};
-            this.scssCompiler(compilerOptions, doneCompiling);
+            this.scssCompiler(compilerOptions, (err, css) => {
+                if (err) {
+                    return callback(err);
+                }
+
+                callback(null, this.autoPrefix(css, options.autoprefixerOptions));
+            });
             break;
         }
+    };
 
-        function doneCompiling(err, css) {
-            if (err) {
-                return callback(err);
-            }
-
-            const prefixedCss = autoprefixer(options.autoprefixerOptions).process(css).css;
+    this.autoPrefix = (css, autoprefixerOptions) => {
+        const payload = typeof css === 'string' || css instanceof Buffer ? css : '';
+        var output = css;
 
-            callback(null, prefixedCss);
+        try {
+            output = postcss([autoprefixer(autoprefixerOptions)]).process(payload).css;
```

Comment:
do we need postcss-sass here to process the payload?  https://github.com/postcss/postcss-scss